e2e/external_dep/BUILD

```diff
@@ -6,7 +6,9 @@ load("@bazel_skylib//rules:build_test.bzl", "build_test")
 ts_project(
     name = "lib",
     srcs = ["lib.ts"],
-    declaration = True,
+    # Composite to also check the location of .tsbuildinfo
+    # https://github.com/aspect-build/rules_ts/issues/329
+    composite = True,
```

Comment:
- I have manually verified that this fails without the fix.
- In a first version, I've added another target here (`lib_composite`). But since `composite` implies declaration, keeping a single one and making the test stronger seemed fine. LMK if you want me to change that.

Reply:
thanks!